cmd/snap-confine/snap-confine.c

```diff
@@ -801,6 +838,8 @@ static void enter_non_classic_execution_environment(sc_invocation *inv, struct s
     // starting the snap application has already ensure that the process has
     // been put in a dedicated group.
     if (!sc_cgroup_is_v2()) {
+        // Temporarily raise egid so we can chown the freezer cgroup
```

Comment:
Is this a TODO? A comment that was missing? It looks out-of-place.

Reply:
dropped the comment, no longer relevant